docs/src/systems/fluid.md

```diff
@@ -19,71 +20,132 @@ Pages = [joinpath("general", "corrections.jl")]
+
+**Key Features:**
+- Particles within half the support radius experience a repulsive force to prevent clustering.
+- Particles beyond half the radius but within the support radius experience an attractive force to simulate cohesion.
+
+Mathematically:
 ```math
 F_{\text{cohesion}} = -\sigma m_b C(r) \frac{r}{\Vert r \Vert},
 ```
-where:
-- ``\sigma`` represents the surface tension coefficient, adjusting the overall strength of the cohesion effect.
-- ``C`` is a scalar function of the distance between particles.
-
-The cohesion kernel ``C`` is defined as
+where \( C(r) \), the cohesion kernel, is defined as:
 ```math
 C(r)=\frac{32}{\pi h_c^9}
 \begin{cases}
-(h_c-r)^3 r^3, & \text{if } 2r > h_c \\
-2(h_c-r)^3 r^3 - \frac{h^6}{64}, & \text{if } r > 0 \text{ and } 2r \leq h_c \\
-0, & \text{otherwise}
+(h_c-r)^3 r^3, & \text{if } 2r > h_c, \\
+2(h_c-r)^3 r^3 - \frac{h^6}{64}, & \text{if } r > 0 \text{ and } 2r \leq h_c, \\
+0, & \text{otherwise.}
 \end{cases}
 ```
 
-#### Surface area minimization force
-To model the minimization of the surface area and curvature of the fluid, a curvature force is used, which is calculated as
+#### Surface Area Minimization Force
+
+The surface area minimization force models the curvature reduction effects, aligning particle motion to reduce the interface's total area. It acts based on the difference in surface normals:
 ```math
-F_{\text{curvature}} = -\sigma (n_a - n_b)
+F_{\text{curvature}} = -\sigma (n_a - n_b),
 ```
+where \( n_a \) and \( n_b \) are the surface normals of the interacting particles.
+
+#### Wall Adhesion Force
 
-#### Wall adhesion force
-The wall adhesion model proposed by Akinci et al. is based on a kernel function which is 0 from 0.0 to 0.5 support radiia with a maximum at 0.75.
-With the force calculated with an adhesion coefficient ``\beta`` as
+This force models the interaction between fluid and solid boundaries, simulating adhesion effects at walls. It uses a custom kernel with a peak at 0.75 times the support radius:
 ```math
 F_{\text{adhesion}} = -\beta m_b A(r) \frac{r}{\Vert r \Vert},
 ```
-with ``A`` being the adhesion kernel defined as
+where \( A(r) \) is the adhesion kernel:
 ```math
-A(r)= \frac{0.007}{h_c^{3.25}}
+A(r) = \frac{0.007}{h_c^{3.25}}
 \begin{cases}
-\sqrt[4]{- \frac{4r^2}{h_c} + 6r - 2h_c}, & \text{if } 2r > h_c \text{ and } r \leq h_c \\
+\sqrt[4]{-\frac{4r^2}{h_c} + 6r - 2h_c}, & \text{if } 2r > h_c \text{ and } r \leq h_c, \\
 0, & \text{otherwise.}
 \end{cases}
 ```
 
+---
+
+### Morris-Based Momentum-Conserving Surface Tension Model
+
+In addition to the Akinci model, Morris (2000) introduced a momentum-conserving approach to surface tension. This model uses stress tensors to ensure exact conservation of linear momentum, providing a robust method for high-resolution simulations.
+
+#### Stress Tensor Formulation
+
+The force is calculated as:
+```math
+F_{\text{surface tension}} = \nabla \cdot S,
```

Comment:
Would it make sense to add another equation explaining how the divergence is computed numerically?